hack/generate-yamls.sh

```diff
@@ -52,6 +52,12 @@ rm -fr ${YAML_OUTPUT_DIR}/*.yaml
 # Generated Knative Operator component YAML files
 readonly OPERATOR_YAML=${YAML_OUTPUT_DIR}/operator.yaml
 
+if [[ -n "${TAG}" ]]; then
+  LABEL_YAML_CMD=(sed -e "s|operator.knative.dev/release: devel|operator.knative.dev/release: \"${TAG}\"|")
```

Comment:
Don't you want $() here?

Reply:
Ah, I think I get it now. I'm not familiar with the () style for constructing strings. neat